apps/condo/domains/miniapp/tasks/deleteB2BAppRoles.js

```diff
@@ -0,0 +1,46 @@
+const dayjs = require('dayjs')
+const chunk = require('lodash/chunk')
+
+const { find, getSchemaCtx } = require('@open-condo/keystone/schema')
+const { createTask } = require('@open-condo/keystone/tasks')
+
+const { B2BAppRole } = require('@condo/domains/miniapp/utils/serverSchema')
+
+// Condo allow 1000 objs per response, including relations
+// { app { id } role { id } permissions ${COMMON_FIELDS} } is 3 objects per 1
+// So the upper limit is 333, but we'll use the number 100 as a reserve
```

Comment:
it is unlikely that there will be so many roles in organization :)

Reply:
:)